mllib/src/test/scala/org/apache/spark/ml/classification/OneVsRestSuite.scala

```diff
@@ -223,6 +223,13 @@ class OneVsRestSuite extends MLTest with DefaultReadWriteTest {
     assert(oldCols === newCols)
   }
 
+  test("SPARK-SPARK-34356: OneVsRestModel.transform should avoid potential column conflict") {
```

Comment:
this test will fail in master and (maybe) all version of OVR.
but I think fix it in master maybe enough.